src/bootstrap/src/core/builder/tests.rs

```diff
@@ -1294,6 +1294,38 @@ mod snapshot {
         ");
     }
 
+    #[test]
+    fn check_cross_compile() {
+        let ctx = TestCtx::new();
+        insta::assert_snapshot!(
+            ctx.config("check")
+                .stage(2)
+                .targets(&[TEST_TRIPLE_1])
+                .hosts(&[TEST_TRIPLE_1])
```

Comment:
Nit (tiny, just for consistency): can you swap `targets()` and `hosts()` around? Usually the ordering is hosts then targets right :D